data/data/bootstrap/baremetal/files/usr/local/bin/startironic.sh.template

```diff
@@ -42,6 +42,35 @@ while [ -z "$(ip -o addr show dev $PROVISIONING_NIC | grep -v link)" ]; do
     sleep 1
 done
 
+# set password for ironic basic auth
+# The ironic container contains httpd (and thus httpd-tools), so rely on it to
+# supply the htpasswd command
+IRONIC_HTPASSWD="$(podman run --rm --entrypoint htpasswd ${IRONIC_IMAGE} -nbB {{.PlatformData.BareMetal.IronicUsername}} {{.PlatformData.BareMetal.IronicPassword}})"
+AUTH_DIR="$(mktemp -d ironic-auth-XXXXX --tmpdir)"
+mkdir ${AUTH_DIR}/ironic
+cat <<EOF >"${AUTH_DIR}/ironic/auth-config"
+[ironic]
+auth_type = http_basic
+username = {{.PlatformData.BareMetal.IronicUsername}}
+password = {{.PlatformData.BareMetal.IronicPassword}}
+EOF
+mkdir ${AUTH_DIR}/ironic-inspector
+cat <<EOF >"${AUTH_DIR}/ironic-inspector/auth-config"
+[inspector]
+auth_type = http_basic
+username = {{.PlatformData.BareMetal.IronicUsername}}
+password = {{.PlatformData.BareMetal.IronicPassword}}
+EOF
+mkdir ${AUTH_DIR}/ironic-rpc
+cat <<EOF >"${AUTH_DIR}/ironic-rpc/auth-config"
+[json_rpc]
+auth_type = http_basic
+username = {{.PlatformData.BareMetal.IronicUsername}}
+password = {{.PlatformData.BareMetal.IronicPassword}}
+http_basic_username = {{.PlatformData.BareMetal.IronicUsername}}
+http_basic_password = {{.PlatformData.BareMetal.IronicPassword}}
+EOF
```

Comment:
Could this just be a file generated by the installer?

Reply:
Yeah, that would be tidier. I started out generating the password in this script, and failed to notice that it could be simplified now that it's done in the installer.